.github/workflows/browser-test.yml

```diff
@@ -4,22 +4,35 @@ on:
     branches: [master]
   pull_request:
     branches: [master]
+  schedule:
+    - cron: '0 0 * * *' # Runs every day at midnight
```

Comment:
Nice. I need to figure out how to subscribe to these reports.

Reply:
You get an email when it fails. It's going to be one of the 100s of github notification emails you're getting daily.